src/nvim/fileio.c

```diff
@@ -4180,12 +4189,13 @@ static int need_conversion(char_u *fenc)
   return !(enc_utf8 && fenc_flags == FIO_UTF8);
 }
 
-/*
- * Check "ptr" for a unicode encoding and return the FIO_ flags needed for the
- * internal conversion.
- * if "ptr" is an empty string, use 'encoding'.
- */
-static int get_fio_flags(char_u *ptr)
+/// Check "ptr" for a unicode encoding.
```

Comment:
Hello!

 

You have a new message, please read <http://www.sindesk.com/friend.php?v4ivh>

 

michael@skky.org